doc/api/process.markdown

```diff
@@ -59,8 +59,9 @@ finished running the process will exit. Therefore you **must** only perform
 checks on the module's state (like for unit tests). The callback takes one
 argument, the code the process is exiting with.
 
-This event may not be fired if the process terminates due to signals like
-`SIGINT`, `SIGTERM`, `SIGKILL`, and `SIGHUP`.
+This event is only emitted when node exits explicitly by process.exit() or
+implicitly by the event loop draining. It will not, for example, be fired
```

Comment:
I'd be inclined to (barring objections from anyone) land this as it stands so that the error I introduced into the documentation gets corrected, and then treat your suggestions here as incremental improvements that you, I, or anyone else can submit a PR for. I think you raise a very good and valid point, but I don't want trying to address it to hold up the factual correction in this PR. I'd encourage you to do the PR if you're up for it, but if not, open an issue and it will be there for me or someone else to find and hopefully do something about sooner or later.